usaspending_api/api_docs/api_documentation/federal_account/spending_over_time.md

```diff
@@ -0,0 +1,63 @@
+## Federal Account Spending Over Time - Deprecated
```

Comment:
This route is not used, remove from docs AND API

Reply:
done